packages/adapter-flagsmith/src/index.ts

```diff
@@ -0,0 +1,84 @@
+import type { Adapter } from 'flags';
+import flagsmith, {
+  IFlagsmithFeature,
+  IInitConfig,
+  IIdentity,
+  createFlagsmithInstance,
+} from 'flagsmith';
+
+type AdapterParams = {
+  key?: string;
+  projectID?: string;
+  dashboardURL?: string;
+} & IInitConfig;
+
+export function createFlagsmithAdapter<
+  ValueType extends IFlagsmithFeature,
+  EntitiesType extends IIdentity,
+>({
+  key: customKey,
+  environmentID,
+  projectID,
+  dashboardURL,
+  ...configParams
+}: AdapterParams): Adapter<ValueType, EntitiesType> {
+  async function initialize() {
+    const { identity, ...restConfigParams } = configParams;
+    const flagsmithInstance = createFlagsmithInstance();
+    await flagsmithInstance.init({ fetch: global.fetch, ...restConfigParams });
```

Comment:
```suggestion
    await flagsmithInstance.init({ fetch: globalThis.fetch, ...restConfigParams });
```

`global` is not defined